stable/kapacitor/README.md

```diff
@@ -0,0 +1,92 @@
+# Kapacitor
+
+##  An Open-Source Time Series ETL and Alerting Engine
+
+[Kapacitor](https://github.com/influxdata/kapacitor) is an open-source framework built by the folks over at [InfluxData](https://influxdata.com) and written in Go for processing, monitoring, and alerting on time series data 
+
+## QuickStart
+
+```bash
+$ helm install stable/kapacitor --name foo --namespace bar
+```
+
+## Introduction
+
+This chart bootstraps A Kapacitor deployment and service on a Kubernetes cluster using the Helm Package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+
+- PV provisioner support in the underlying infrastructure (optional)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/kapacitor
+```
+
+The command deploys Kapacitor on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release --purge
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the Kapacitor chart and 
+their default values.
+
+```yaml
```

Comment:
Rather than listing what they can check themselves, I'd suggest yanking the config tables we've been using in some of the other charts that describe the purpose of each param. We can defer to values.yaml for the default values. This can help make it easier to update these in the futures as well, since we end up with one source of truth to update instead of two.